boards/nxp/mimxrt1160_evk/mimxrt1160_evk.dtsi

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "mimxrt1160_evk-pinctrl.dtsi"
+#include <nxp/nxp_rt1160.dtsi>
```

Comment:
just curious why was this not here before and why is it being added now? (it's not a problem I'm just wondering what happened)